src/interfaces.ts

```diff
@@ -19,6 +19,8 @@ interface IFieldConfigBase {
   key: string;
   label: string | null;
   nullify: boolean;
+  populateFromSearch: boolean;
```

Comment:
Could there be an addition to the documentation about what these do? The names are relatively clear but could be confusing to the unacquainted

Reply:
Good point. Written.